README.md

```diff
@@ -3,10 +3,10 @@
 The RDF/JS SHACL-UI Task Force is a focus group of the RDF/JS Community Group.
 
 The objective is to enhance the SHACL ontology to cater specifically to the UI use cases for application development and data exploration.
-It covers reading and writing RDF data.
+It covers viewing/displaying, writing and querying RDF data.
```

Comment:
I would stay with reading/writing or viewing/editing but not mix them.

What do you think about changing querying to filtering? Querying could also be the process of fetching options for a dropdown for a form. I would connect filtering with faceted search, and I guess that's what you mean.

Reply:
Yes, OK for viewing, editing and filtering. Makes sense. I am updating accordingly.